### ISSUE TITLE
glutathionuria

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9648]
**Preferred term label:**
glutathionuria

**Synonyms**

**Definition (free text, please give PubMed ID)**
 increased level of glutathione in the urine.
PMID:29483667
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
231950 
PMID: 29483667   2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
